Classes/Network/USRExecutor.swift

```diff
@@ -9,16 +9,14 @@
 import UIKit
 
 
-public class USRExecutor :NSObject,RequestExecutor, URLSessionDelegate {
-    
+public class USRExecutor: NSObject, RequestExecutor, URLSessionDelegate {
     
     var tasks: [URLSessionDataTask] = [URLSessionDataTask]()
-    var taskIdByRequestID: [String:Int] = [String:Int]()
+    var taskIdByRequestID: [String : Int] = [String : Int]()
```

Comment:
I know what you think, but the convention is `[String: Int]`, not `[String : Int]`. This is also consistent with variable declaration: `value: Int`.

Consider this:

```swift
var dict: [String: Int]
var number: Int
```

- `number` is a variable whose value is an `Int`.
- `dict` is a dictionary whose values are `Int`s.